### PR TITLE
Update mustache-d to upstream repo

### PR DESCRIPTION
**Pull request by wilzbach**
No need to maintain our own fork anymore and [repo](https://github.com/repeatedly/mustache-d) seems more active lately.

### CODE REVIEW

File: dub.sdl [hunk 1]
```diff
@@ -6,7 +6,7 @@ homepage "http://tour.dlang.org"
 license "Boost"
 dependency "vibe-d" version="~>0.7.31-rc.3"
```
Comment (ZombineDev) [inline dub.sdl]: Can you also update vibe.d to 0.7.31, i.e. non-rc? (I'm not sure if vibe.d 0.8 is ready fo production, Sonke mentioned at dconf that there were some performance issues that he needed to investigate.)

Comment (wilzbach) [inline dub.sdl]: No it's not. Sure will do later. Thanks for noticing

**Comment by ZombineDev** [comment]
BTW, dmd-nightly produces a link-time error that I need to properly reduce

**Comment by wilzbach** [comment]
Thanks a lot @ZombineDev

**Pull request merged**